### PR TITLE
Groups Contact

### PR DESCRIPTION
**Pull request by JakeDawkins**
# Feature / Fixed Issue(s)
- Added inline enhancer for pulling phone numbers for usage in the join group modal.

### CODE REVIEW

**Comment by jbaxleyiii** [comment]
**[Storybook Hub](https://hub.getstorybook.io/)** is building storybooks for every commit in this PR.

**Latest Storybook: https://2b0b4398-329a-498f-a0d3-bb2206677731.sbook.io/**
<sub>( Visit [here](https://hub.getstorybook.io/app/github.com/NewSpring/Holtzman/branch/phone-numbers/) for more info )</sub>

> _This comment is automatically added by Storybook Hub_

File: imports/pages/groups/profile/index.js [hunk 1]
```diff
@@ -19,6 +19,24 @@ import { modal } from "../../../data/store";
 import Layout from "./Layout";
 import Join from "./Join";
 
+const PHONE_QUERY = gql`
```
Comment (jbaxleyiii) [inline imports/pages/groups/profile/index.js]: @jakedawkins let's name this query so we can update it later

File: imports/pages/groups/profile/index.js [hunk 2]
```diff
@@ -19,6 +19,24 @@ import { modal } from "../../../data/store";
...
+  currentPerson {
+    phoneNumbers {
+      number
+      rawNumber
+    }
+  }
+}`;
+
+export const JoinWithPhones = graphql(PHONE_QUERY, {
+  name: "phoneNumbers",
```
Comment (JakeDawkins) [inline imports/pages/groups/profile/index.js]: like dis, @jbaxleyiii?

Comment (jbaxleyiii) [inline imports/pages/groups/profile/index.js]: @JakeDawkins like this:

```js
const PHONE_QUERY = gql`
query PhoneNumbers {
 ...
}`;
```

That way @richarddubay can use http://dev.apollodata.com/react/cache-updates.html#updateQueries

**Comment by jbaxleyiii** [comment]
@JakeDawkins this looks great. Lets hold off on merging into master until the feature is complete from @richarddubay.  @richarddubay I'd recommend pointing your branch into this one and rebasing

**Comment by jbaxleyiii** [comment]
@JakeDawkins was this part of @richarddubay's PR ( #1814 )?

**Comment by richarddubay** [comment]
@jbaxleyiii I added the `withGroupMutation(withAddPhoneNumber` bit, but not the `decorate(Shareable)`. So...it's both of ours? cc: @JakeDawkins

**Pull request merged**